tests/Backend/MixedTeradata/SharingTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+// no SNFLK/other backend interaction because it is disabled for time being
+namespace Keboola\Test\Backend\MixedTeradata;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Keboola\Test\Backend\Mixed\StorageApiSharingTestCase;
+use Keboola\Test\Backend\WorkspaceConnectionTrait;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+
+class SharingTest extends StorageApiSharingTestCase
+{
+    use WorkspaceConnectionTrait;
+
+    /**
+     * @return void
+     */
+    public function testOrganizationAdminInTokenVerify(): void
+    {
+        $token = $this->_client->verifyToken();
+        self::assertTrue($token['admin']['isOrganizationMember']);
+    }
```

Comment:
proč je to vlastní test? TO by mělo byt v setupu, ne? Aby sa ti nepustil žádný ten test case.

Reply:
![image](https://user-images.githubusercontent.com/180148/200584614-8b98698f-2742-46de-af7a-45fe8d9e41a9.png)

Reply:
zrusil jsem to ze vsech mixed suit. Dela se to uz v setUpu, jen jsem tam presunul assert